test/e2e/specs/highlightsGraphPageVideoIsPlaying.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { vpTest } from '../fixtures/vpTest';
+import { expect, test } from '@playwright/test';
+import { waitForPageToLoadWithTimeout } from '../src/helpers/waitForPageToLoadWithTimeout';
+import { ExampleLinkType } from '../types/exampleLinkType';
+import { HighlightsGraphPage } from '../src/pom/highlightsGraphPage';
+
+// Link to AI Highlights Graph page
+const link: ExampleLinkType = { name: 'AI Highlights Graph', endpoint: 'highlights-graph.html' };
```

Comment:
you already have an array of those links objects, write a function that returns link type by name instead of 
{ name: 'AI Highlights Graph', endpoint: 'highlights-graph.html' };

Reply:
getLinkByName function was added and also the support enum for example page names